src/AirPyllution/airpyllution.py

```diff
@@ -67,4 +67,31 @@ def get_air_pollution(lat, lon, api_key):
     Examples
     --------
     >>> get_air_pollution(49.2497, -123.1193, "APIKEY_example")
+    """
+
+def get_pollution_forecast(lat, lon, api_key):
+    """Returns a time series plot showing predicted pollutant levels for the next 5 days
```

Comment:
Forgot a period at the end lol

Reply:
Do you want to me to add ` for the next 5 days from the current date ` ?

Reply:
Yah I meant a literal period `.`

Reply:
Done.